Source/monster.cpp

```diff
@@ -50,6 +50,19 @@ void NewMonsterAnim(int i, AnimStruct *anim, Direction md, AnimationDistribution
 	monst->_mdir = md;
 }
 
+void StartMonsterGotHit(int mid)

```

Comment:
Also if passing the reference is not possible, consider creating a local `monster` variable that assigns the reference to avoid duplication.

Reply:
done 🙂